src/webgl/p5.RendererGL.js

```diff
@@ -1104,6 +1111,7 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
         this._pInst.noStroke();
         this._pInst.blendMode(constants.BLEND);
         this._pInst.shader(this.filterShader);
+        this.filterShader.setUniform('uNewNormalMatrix', this.uNMatrix.mat3);
```

Comment:
Since this only is used in the sphere mapping shader, can we do this and the `inverseTranspose` line in `getSphereMapping` where we're setting another uniform already?